crypto/db/db.go

```diff
@@ -229,6 +229,49 @@ func (db *Mgd) GetKeyValueFromBucket(
 	return found, nil
 }
 
+// GetAllValuesFromBucket returns all entries from the bucket.
+// Read function can be used to decrypt the data.
+// Errors will return only if it cannot perform the transaction successfully.
+func GetAllValuesFromBucket(
+	bucket []byte,
+	read Filter,
```

Comment:
I almost forgot this function would be variadic as well if the called one would be.